data/persons.txt

```diff
@@ -2035,3 +2035,179 @@ outfit "Emotional Baggage"
 	thumbnail "outfit/unknown"
 	description `The physical embodiment of everything that makes the Reviewers who they are today.`
 	description `	Unfortunately, it's not for sale.`
+
+
+
+person "Danau"
+	government "Author"
+	frequency 100
+	personality
+		heroic unconstrained opportunistic
+	phrase
+		word
+			"When you want to stop your ship, then press shift and back."
```

Comment:
This sentence confuses me. I get what it's trying to say, but I'm not sure it's working.

`then` would work better with `if` than with `when`. You could probably remove it entirely.

Back isn't a key, but it makes sense in case as a verb if you've rebound your keys. Still, I'd probably go with "reverse" or similar, since that use of back is very niche.

```suggestion
			"When you want to stop your ship, press shift and reverse."
```